bedrock/thunderbird/templates/thunderbird/all.html

```diff
@@ -0,0 +1,126 @@
+{% extends "thunderbird/base-resp.html" %}
```

Comment:
This template seems to share a lot with `firefox/all.html`. Is it possible that we could make a useful base template for all of these pages?